src/site/layouts/bios_page.drupal.liquid

```diff
@@ -43,7 +43,7 @@
 
             {% include 'src/site/navigation/facility_sidebar_nav.drupal.liquid' with sidebarData = facilitySidebar %}
 
-            <div class="usa-width-three-fourths">
+            <div class="usa-width-two-thirds">
```

Comment:
It looks like this will affect the side-navs on pages in production (so beyond the React components we're building.) Is that okay?

Reply:
That's what I warned about in the PR description and I commented on the issue about this. I don't think this should be merged until we're ready to have the new SideNav on production tbh!